packages/next/src/compiled/@edge-runtime/cookies/index.js

```diff
@@ -28,6 +28,87 @@ __export(src_exports, {
 });
 module.exports = __toCommonJS(src_exports);
 
+
+/**
```

Comment:
This code addition is unexpected 🤔

Reply:
Will fix via https://github.com/vercel/next.js/pull/57082